### PR TITLE
add: admin field/audit log mappings

### PR DESCRIPTION
**Pull request by ericpaulsen**
adds tables for the expected outputs of admin setting changes in the audit log.

### CODE REVIEW

File: admin/audit.md [hunk 1]
```diff
@@ -58,3 +58,75 @@ actions included:
 - `view`: the Coder CLI used a secret
 - `write`: the user made a change to a Coder entity (e.g., workspace, user,
   resource pool, etc.)
+
+## Admin logged events
+
+With the exception of a few, logged events made by Admin panel changes will output
```
Comment (bpmct) [inline admin/audit.md:64]: Should we document the exception?

**Review by bpmct — APPROVED** [review]
This is super thorough! Awesome addition to the docs.

Do we have a plan for how we update this when new features are added? It may be infrequent, but perhaps code comments or something would help us remember to update this?

File: admin/audit.md (hunk 1 above)
Comment (ericpaulsen) [reply admin/audit.md:64]: we should, but those will be added soon, so we'd have to go back and remove it. i'm thinking whatever is not documenting isn't logged. what do you think?

Comment (bpmct) [reply admin/audit.md:64]: that's fair.

**Pull request merged**